utils/api/export.ts

```diff
@@ -202,7 +201,7 @@ ${yaml.stringify(frontmatter)}
 					const data = await res.blob();
 					const fileType = data.type.split("/").pop();
 					attachmentsFolder.file(`${fileName}.${fileType}`, data);
-					md.replaceAll(
+					md = md.replaceAll(
```

Comment:
In JavaScript, strings are immutable - meaning their values cannot be changed once created. All string methods like replaceAll() will return a new string, leaving the original string untouched.